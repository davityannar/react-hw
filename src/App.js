import React, {Component} from 'react';

import { Route, Switch, withRouter } from 'react-router-dom'
import MainPage from './mainComponents/MainPage/MainPage';
import Homepage from './mainComponents/MainPage/MainPage';

import 'isomorphic-fetch';
import 'babel-polyfill';


const PageNotFound = () => (
    <div>
        <h1> 404 Page Not Found </h1>
    </div>
);

export class App extends Component {

    constructor(props) {
        super(props)
      }


   
    render() {
  

        return (   
          <Switch>
            <Route exact path='/' component={Homepage} />
            <Route path={['/movies/:id', '/movies']} component={MainPage} />
            <Route component={PageNotFound} />
          </Switch>
         
            
                        
        )
    }
}
export default withRouter(App)