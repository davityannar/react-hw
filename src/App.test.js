import React from 'react';
import { shallow } from 'enzyme';
import App from './App';

describe('App', () => {
  it('Adds App snapshot testing', () => {
    const component = shallow(<App />);

    expect(component).toMatchSnapshot();
  });
});