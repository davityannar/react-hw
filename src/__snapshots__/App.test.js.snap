// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App Adds App snapshot testing 1`] = `
<ErrorBoundary>
  <div>
    <Header />
    <SortingSection />
    <Card />
    <Footer />
  </div>
</ErrorBoundary>
`;
